src/applications/simple-forms/form-upload/config/constants.js

```diff
@@ -124,3 +134,55 @@ export const ADD_CHILD_CONTENT_0779 = Object.freeze(
     </div>
   </>,
 );
+
+export const ALERT_TOO_MANY_PAGES = (formNumber, onCloseEvent) =>
+  Object.freeze(
+    <VaAlert
+      close-btn-aria-label="Close notification"
+      status="warning"
+      visible
+      closeable
+      onCloseEvent={onCloseEvent}
+    >
+      <h2 slot="headline">
+        Are you sure the file you uploaded is VA Form {formNumber}?
+      </h2>
+      <React.Fragment key=".1">
+        <p className="vads-u-margin-y--0">
+          The file you uploaded has more pages than the form usually has. Please
+          check the file you uploaded is a recent VA Form {formNumber}.
+        </p>
+        <a href={DOWNLOAD_URL_0779}>
+          Download VA Form {formNumber}
+          (PDF)
+        </a>
+        <p>If you’re sure this is the right file, you can continue.</p>
+      </React.Fragment>
+    </VaAlert>,
+  );
+
+export const ALERT_TOO_FEW_PAGES = (formNumber, onCloseEvent) =>
+  Object.freeze(
+    <VaAlert
+      close-btn-aria-label="Close notification"
+      status="warning"
+      visible
+      closeable
+      onCloseEvent={onCloseEvent}
+    >
+      <h2 slot="headline">
+        Are you sure the file you uploaded is VA Form {formNumber}?
+      </h2>
+      <React.Fragment key=".1">
+        <p className="vads-u-margin-y--0">
+          The file you uploaded has fewer pages than the original form. Please
+          check your uploaded form to be sure it is the correct form.
+        </p>
+        <a href={DOWNLOAD_URL_0779}>
```

Comment:
Everything else in this object is conditional on the formNumber but this is hardcoded to be 21-0779 every time.